.github/workflows/ckan.yml

```diff
@@ -10,7 +10,6 @@ on:
         type: choice
         options:
           - development
-          - staging
```

Comment:
did we remove staging just to speed up runs?

Reply:
nah. just to stop anyone from manually running a GH action when the DB is supposed to be frozen during upgrade.